eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdout.java

```diff
@@ -55,14 +55,24 @@ public class EOstdout extends PhDefault {
      * @param sigma Sigma
      */
     public EOstdout(final Phi sigma) {
+        this(sigma, EOstdout.OUT);
+    }
+
+    /**
+     * Ctor.
+     * It's supposed to used only for the test purposes.
+     * @param sigma Sigma
+     * @param out Output stream.
+     */
+    public EOstdout(final Phi sigma, final PrintStream out) {
```

Comment:
@Graur yes, I tried but had no luck. Tests run in parallel and since `outContent` is static (one for all tests) - it is being rewriting and resetting many times randomly.

Reply:
@yegor256 Can you help us to understand if creating a new `EOstdout` constructor just for test cases is considered a valid case?

Reply:
@yegor256 please have a look

Reply:
@yegor256 please have a look